packages/connect/src/api/nem/api/nemSignTransaction.ts

```diff
@@ -19,6 +19,11 @@ export default class NEMSignTransaction extends AbstractMethod<
 
         const { payload } = this;
         // validate incoming parameters
+
+        // Workaround for NEM timestamp case-sensitivity issue
+        if ((payload.transaction as any).timestamp) {
```

Comment:
in case not yet validation payload.transaction is undefined, this would throw, wouldn't it?

Reply:
True, fixing now